### ISSUE TITLE
Use react-router basename

### ISSUE THREAD

**Issue by thangngoc89** [MoOx/statinamic#115]
This would allow me to proceed #67 further.
Still not sure how to do this in react-router@2.0.0-rcx

tracking this https://github.com/rackt/react-router/issues/3045

---
Edit: 

react-router@1.0.0 api:

```js
import { useRouterHistory } from 'react-router'
import { createHistory, useBasename } from 'history'

const browserHistory = useRouterHistory(useBasename(createHistory))({
  basename: "/statinamic"
});
```

react-router@2.0.0

https://github.com/rackt/react-router/blob/master/modules/useRouterHistory.js

**Comment by MoOx** [comment]
Just for the record https://github.com/rackt/react-router/issues/2014

**Comment by thangngoc89** [comment]
This require more work than I thought.

**Comment by thangngoc89** [comment]
Didn't aware of this. 
https://github.com/reactjs/react-router/pull/3054/files#diff-db41db2de1c0357baa6227d912609546R156

Should be simple right now

**Comment by thangngoc89** [comment]
Merged in 0.8.0

**Comment by MoOx** [comment]
Closed by bdad71a29e7126ed511fbba4425c5af285a63926
